uaclient/security_status.py

```diff
@@ -63,6 +64,22 @@ def get_origin_information_to_service_map():
     }
 
 
+@lru_cache(maxsize=None)
+def get_esm_cache():
+    try:
+        # If the rootdir folder doesn't contain any apt source info, the
+        # cache will be empty
+        cache = Cache(rootdir=ESM_APT_ROOTDIR)
+    except Exception:
+        cache = {}
+
+    result = {}
```

Comment:
You don't need this step - the cache itself works a Dict {pkg_name: pkg}. Returning the cache here should be fine.